README.md

```diff
@@ -75,6 +75,11 @@ the binary verification command above.
 
 ## Build
 
+To quickly test that io.js was built correctly from any of the steps below:
```

Comment:
@jbergstroem, good point. I've updated the pull request with a new commit that moves it to the bottom.